Dockerfile.hpu.ubi

```diff
@@ -82,12 +82,15 @@ RUN dnf install -y --setopt=install_weak_deps=0 --nodocs gcc \
 # install vllm wheel first, so that torch etc will be installed
 RUN --mount=type=bind,from=build,src=/workspace/dist,target=/workspace/dist \
     --mount=type=cache,target=/root/.cache/pip \
-    pip install $(echo dist/*.whl)'[tensorizer]' --verbose
+    pip install $(echo dist/*.whl)'[tensorizer]' jinja2>=3.1.5 --verbose
 
 ENV HF_HUB_OFFLINE=1 \
     PORT=8000 \
     HOME=/home/vllm \
-    VLLM_USAGE_SOURCE=production-docker-image
+    VLLM_USAGE_SOURCE=production-docker-image \
+    VLLM_NO_USAGE_STATS=1 \
+    VLLM_SKIP_WARMUP=true \
+    PT_HPU_MAX_COMPOUND_OP_SIZE=10
```

Comment:
These changes seem unrelated (CVE fix + other). Can you split it up in separate commits and update the PR title accordingly?

Reply:
done... create three seperate commit